v0.3/cruorg/components/searchbar/searchbar.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+function output_searchbar($searchbar_type)
+{
+?>
+  <div style="padding: 8px; <?php if ($searchbar_type == 'black') echo ' background-color: #383F43;' ?>">
```

Comment:
@sebamelzi Don't include HTML for design preview purposes in the functions. Only AEM markup should be in these.